ONSdigital/census-rh-ui
# CR-1241 Call Centre Contact Numbers

Signed-off-by: Paul-Joel <paul.joel@ons.gov.uk>

# Motivation and Context
Update to make Call Centre telephone numbers individual, and change a page title

# What has changed
New function to return appropriate telephone number when given a display_region
Update to call centre page to use new function
Update to paper form manager page to use new function
Update to existing unit tests to add check for appropriate telephone number
Removal of unused param in some unit tests
New test to check new function
Update of page title when user selects 'I cannot find my address' in address selection list

# How to test?
Trigger the 'contact the call centre' page for its various trigger points. Switch display_region and confirm the telephone number changes

Tested against RHCucumber - no updates required

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1241

# Screenshots (if appropriate):
